fetch/api/response/multi-globals/url-parsing.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<title>Response.redirect URL parsing, with multiple globals in play</title>
+<link rel="help" href="https://fetch.spec.whatwg.org/#dom-request">
```

Comment:
Probably #dom-response here.
There is also some code shared with request/multi-globals/url-parsing.html
Maybe it would be fine to create a fetch/api/multi-globals folder?